travel-planner/travel-planner-ui/src/Components/Marker/Content.jsx

```diff
@@ -0,0 +1,90 @@
+import "./Marker.css";
+import { useState } from "react";
+import axios from "axios";
+
+// create delete marker button?
+// allow us to edit note content and also expand note content on the right side.
+
+export default function Content({
+	obj,
```

Comment:
What does obj mean? Could we rename it for easier to understand?

Reply:
I can rename this to infoWindow.